### PR TITLE
Improve tests of `captureCheckIn()`

### PR DESCRIPTION
**Pull request by ste93cry**
Round of small improvements to the tests of the `captureCheckIn()` function. I also noticed that the method definition added to `HubInterface` was incorrect, so I fixed it.

### CODE REVIEW

File: src/State/HubInterface.php [hunk 1]
```diff
@@ -23,7 +23,7 @@
  * stack of pairs of clients and scopes. It is the main entry point to talk
  * with the Sentry client.
  *
- * @method string|null captureCheckIn(string $slug, CheckInStatus $status, $duration = null, ?MonitorConfig $upsertMonitorConfig = null, ?string $checkInId = null) Captures a check-in
+ * @method string|null captureCheckIn(string $slug, CheckInStatus $status, int|float|null $duration = null, ?MonitorConfig $monitorConfig = null, ?string $checkInId = null) Captures a check-in
```
Comment (cleptric) [inline src/State/HubInterface.php:26]: Does this make phpstan happier now? Maybe we can update the baseline file.

Comment (ste93cry) [reply src/State/HubInterface.php:26]: The baseline hasn't changed since 2 months, why should this change affect it?

Comment (cleptric) [reply src/State/HubInterface.php:26]: Ahh, I meant we might be able to update the doc blocks to `{@inheritdoc}`.

Comment (ste93cry) [reply src/State/HubInterface.php:26]: It doesn't work, but we can use `{@inheritdoc}` and just add `@param int|float|null $duration` next to it to make PHPStan happier, which is better than the current situation anyway

**Pull request merged**